src/ape/__init__.py

```diff
@@ -3,96 +3,57 @@
 signal.signal(signal.SIGINT, lambda s, f: _sys.exit(130))
 
 import sys as _sys
-from functools import partial as _partial
-from pathlib import Path as _Path
+from functools import partial
 
 from .contracts import _Contract
-from .managers.accounts import AccountManager as _AccountManager
-from .managers.chain import ChainManager as _ChainManager
-from .managers.compilers import CompilerManager as _CompilerManager
-from .managers.config import ConfigManager as _ConfigManager
-from .managers.converters import ConversionManager as _ConversionManager
-from .managers.networks import NetworkManager as _NetworkManager
-from .managers.project import ProjectManager as Project
-from .managers.project import _DependencyManager
-from .plugins import PluginManager as _PluginManager
-from .utils import USER_AGENT
+from .managers import (
+    Project,
+    _account_manager,
+    _chain_manager,
+    _compiler_manager,
+    _config_manager,
+    _conversion_manager,
+    _network_manager,
+    _plugin_manager,
+    _project_manager,
+)
 
 # Wiring together the application
-_data_folder = _Path.home().joinpath(".ape")
 
-plugin_manager = _PluginManager()
+plugin_manager = _plugin_manager
 """Manages plugins for the current project. See :class:`ape.plugins.PluginManager`."""
 
-_DependencyManager.plugin_manager = plugin_manager
-_dependency_manager = _DependencyManager(data_folder=_data_folder)
-
-_ConfigManager.plugin_manager = plugin_manager
-_ConfigManager._dependency_manager = _dependency_manager
-config = _ConfigManager(
-    # Store all globally-cached files
-    data_folder=_data_folder,
-    # NOTE: For all HTTP requests we make
-    request_header={
-        "User-Agent": USER_AGENT,
-    },
-    # What we are considering to be the starting project directory
-    project_folder=_Path.cwd(),
-)
+config = _config_manager
```

Comment:
How about this:
```suggestion
config = ManagerAccessBase.config_manager
```

Reply:
I think by pulling it from the `MangagerAccessBase` it possibly won't be initialized, vs. pulling it from the `managers/__init__.py` file?

Reply:
If you pull like `from managers import ManagerAccessBase as _ManagerAccessBase` and then add a note about the hack to ensure it is properly initialized, I think it will work.